app/views/employees/_form.html.erb

```diff
@@ -13,13 +13,13 @@
 
 <div class="field-group field-group--two-cols">
     <div class="field field--required">
-        <%= f.label :last_name, class: "field__label" %>
-        <%= f.text_field :last_name, classs: "field__input" %>
+        <%= f.label :surname, class: "field__label" %>
+        <%= f.text_field :surname, classs: "field__input" %>
```

Comment:
This isn't to do with your code, but I've just noticed that the input `class` symbol is mispelt here and for the forename, and possibly elsewhere too (3 esses, `classs`). I'm curious whether the input HTML it generates actually has that class, or is it just ignoring it because of the typo? Will fixing the typo change the styles? 😄